MobileClient/Client/Components/HorizontalProductScroller.razor

```diff
@@ -1,14 +1,14 @@
 ﻿@inherits BaseComponent
 
 <div class="@Class" style="@Style">
-    <MudButton Variant="Variant.Text" Class="mt-3 ml-1">
-        <p class="h3 my-auto">@Title ></p>
-    </MudButton>
+
+    <p class="h3 my-auto p-2"> @Title ></p>
+
     <div class="itemContainer hideScrollbar">
         @foreach (var item in Items)
         {
             <div class="p-2" @onclick="() => NavigateToList(item.Id)">
-                <div class="item" style="background-image: url(@item.CoverPictureUri); background-size: cover;" />
+            <div class="item" style="background-image: url(@item.CoverPictureUri); background-size: cover; border-color: var(--secondary); border-style:solid; border-width: 2px;" />
                 <div style="text-align: center;">@item.Name</div>
```

Comment:
Wenn du schonmal dabei bist, dann kannst du auch diesen Style in eine "title" Klasse oder so machen.